apps/cyberstorm-remix/cyberstorm/utils/errors/loaderMappings.ts

```diff
@@ -0,0 +1,81 @@
+import type { LoaderErrorMapping } from "./handleLoaderError";
+
+export const SIGN_IN_REQUIRED_MAPPING: LoaderErrorMapping = {
+  status: 401,
+  headline: "Sign in required.",
+  description: "Please sign in to continue.",
+  category: "auth",
+};
+
+export const FORBIDDEN_MAPPING: LoaderErrorMapping = {
+  status: 403,
+  headline: "You do not have permission to perform this action.",
+  description: "Contact a team administrator to request access.",
+  category: "auth",
+};
+
+export const VALIDATION_MAPPING: LoaderErrorMapping = {
+  status: 422,
```

Comment:
I think our backend is currently more likely to return 400 Bad Request than this?